opentelemetry-otlp/tests/integration_test/expected/different_metrics.json

```diff
@@ -0,0 +1,133 @@
+{
```

Comment:
same as `metrics.json` but with minor changes to validate that our comparison is working.